### ISSUE TITLE
macOS 10.13.6 High Sierra, ajour 0.4.4 crashes on start

### ISSUE THREAD

**Issue by robertjamesftw** [casperstorm/ajour#310]
Running an iMac with High Sierra, trying to run ajour 0.4.4, and crashes on startup.

The crash output is very long, attaching the entire report as a text file, but here is the begnning:

      	/Applications/Ajour.app/Contents/MacOS/ajour
Identifier:        	com.rogildstorm.ajour
Version:           	1.0.0 (1)
Code Type:         	X86-64 (Native)
Parent Process:    	??? [1]
Responsible:       	ajour [12360]
User ID:           	501

Date/Time:         	2020-10-26 11:58:47.083 -0400
OS Version:        	Mac OS X 10.13.6 (17G14033)
Report Version:    	12
Anonymous UUID:    	CB1E3B8B-1AF8-B69D-F7F8-79CF3EB5FCD9

Sleep/Wake UUID:   	C7F68C9C-76BC-4387-8AE0-FDFF26075BD8

Time Awake Since Boot: 200000 seconds
Time Since Wake:   	190 seconds

System Integrity Protection: enabled

Crashed Thread:    	0  Dispatch queue: com.apple.main-thread

Exception Type:    	EXC_BAD_INSTRUCTION (SIGILL)
Exception Codes:   	0x0000000000000001, 0x0000000000000000
Exception Note:    	EXC_CORPSE_NOTIFY

Termination Signal:	Illegal instruction: 4
Termination Reason:	Namespace SIGNAL, Code 0x4
Terminating Process:   exc handler [0]

[ajour crash.txt](https://github.com/casperstorm/ajour/files/5440069/ajour.crash.txt)

**Comment by casperstorm** [comment]
It properly relates to #153.
For now it is unknown why, however I have just got my hands on a Mac which has same behavior so I'll start debugging tonight.

I'll keep you updated.

**Comment by Brightbrown** [comment]
I just completed an upgrade to High Sierra, 10.13.6 and got the "immediate crash." I had to re-run it several times before I even got a crash report. Had not been an issue in Sierra 10.12.6. After hints that the issue may lie in fonts, I ran Font Book and downloaded all the fonts that listed gray (was very unsure as it looked like almost all were various Asian fonts). BOOM, app (0.5.1) is now working as expected. My guess is the app may be trying to access some symbol on one of them.
